ui/raidboss/data/06-ew/ultimate/dragonsongs_reprise_ultimate.ts

```diff
@@ -14,7 +14,7 @@ import { LocaleText, TriggerSet } from '../../../../../types/trigger';
 // TODO: "move" call after you take your Brightwing cleave?
 // TODO: Meteor "run" call?
 
-type Phase = 'doorboss' | 'thordan';
+type Phase = 'doorboss' | 'thordan' | 'niddhog' | 'haurchefant' | 'thordan2' | 'niddhog2' | 'dragon-king';
```

Comment:
Sorry for the small typo but `nidhogg` should be spelled like this, here and elsewhere.

Reply:
Sorry again for the small typo, but it's `nidhogg`.

Reply:
Oh man, I can't spell his name. Should have just put Kyle. :snake: